src/display/api.js

```diff
@@ -717,6 +717,8 @@ var PDFDocumentProxy = (function PDFDocumentProxyClosure() {
  * @property {Object} canvasFactory - (optional) The factory that will be used
  *                    when creating canvases. The default value is
  *                    {DOMCanvasFactory}.
+ * @property {string} backgroundColor - (optional) Background color to use for
+ *                    the canvas. The default value is 'rgb(255,255,255)'.
```

Comment:
Have added more docs and changes the name of the parameter to "background" as it's not necessarily a color.